### PR TITLE
Fixed alignment of How To Play for mobile view

### PR DESCRIPTION
**Pull request by muditasisodia**
#37

### CODE REVIEW

File: src/Componenets/Header/Header.css [hunk 1]
```diff
@@ -46,3 +46,13 @@ img {
 .navbar-brand {
   width: 55%;
 }
+
+@media only screen and (max-width: 768px) {
```
Comment (lalitmee) [inline src/Componenets/Header/Header.css]: Hey @muditasisodia, alignment of the nav links is right now. But the logo is becoming a little bit small of resizing. I was wondering if you can fix that too. :innocent: 
screenshot from 2018-04-16 11-04-31

**Comment by muditasisodia** [comment]
image

It seems to be working fine for me, what could be the problem?

**Comment by Vanisha27** [comment]
screenshot
Please have a look at the screenshot. @muditasisodia

**Comment by muditasisodia** [comment]
It seems I made some mistake while committing my changes, let me know if it's okay now

**Comment by lalitmee** [comment]
Hey @muditasisodia, now its okk for the mobile view but I was wondering if you can add media query for this breakpoint also. I think it is around `width: 800px`. :+1: 
screenshot from 2018-04-17 13-37-53

**Comment by muditasisodia** [comment]
image

This is the point just before the nav bar collapses (the smallest that the logo will ever look). Is it okay? I'll push the code after your approval

**Comment by Vanisha27** [comment]
This looks good. :+1: : 
But you'll have to sync your fork with main repo before updating the PR. :innocent:

**Comment by muditasisodia** [comment]
okay, will make the required changes asap

**Pull request merged**